test/Abp.Zero.SampleApp/EntityHistory/Ad.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Abp.Auditing;
+using Abp.Domain.Entities.Auditing;
+
+namespace Abp.Zero.SampleApp.EntityHistory
+{
+    [Audited]
+    public class Ad : FullAuditedEntity
+    {
+        public Ad()
+        {
+        }
+
+        public Ad(string name, string content)
+        {
+            Name = name;
+            Content = content;
+        }
+
+        public string Name { get; set; }
```

Comment:
can we add `DisableAudting` attribute here to simulate the same scenario as `Blog` ?